packages/address-utils/index.ts

```diff
@@ -156,3 +156,8 @@ export function parseDID(did: string) {
 
   return { networkId, assetId, tokenId };
 }
+
+export function getCollectionFromAddress(did: string): string {
```

Comment:
this I guess won't work for Polkadot/hh? Now used to construct NFTscan request so not for Polkadot/hh

Reply:
It should work universally, because all DID addresses are standardized. I checked the codebase, and it should return token/asset ID? If so, we need to rename this function. Also, the parseDID function already does all the deconstruction of the address, but the naming of the variables might be cryptic (web2-web3 compatibility).